docs/upgrade.html

```diff
@@ -79,12 +79,16 @@
     <li> MessageReader's package was changed from <code>kafka.tools</code> to <code>kafka.common</code> </li>
     <li> MirrorMakerMessageHandler no longer exposes the <code>handle(record: MessageAndMetadata[Array[Byte], Array[Byte]])</code> method as it was never called. </li>
     <li> The 0.7 KafkaMigrationTool is no longer packaged with Kafka. If you need to migrate from 0.7 to 0.10.0, please migrate to 0.8 first and then follow the documented upgrade process to upgrade from 0.8 to 0.10.0. </li>
+    <li> The new consumer has standardized its APIs to accept <code>java.util.Collection</code> as the sequence type for method parameters. Existing code will have to be updated and compiled against 0.10.0. </li>
```

Comment:
Maybe it's confusing because we don't emphasize that we're talking about 0.10.0 of the client? Would it help if we rephrased this way: "Existing code will have to be updated to work with the 0.10.0 client library"?